### PR TITLE
Adjust organizations tag description

### PR DESCRIPTION
**Pull request by justnero**

### CODE REVIEW

**Comment by ashkarpetin** [comment]
Please remove deprecated tag https://user-api-docs.rebilly.com/tag/Organizations#operation/GetOrganizationCollection

**Review by ashkarpetin — DISMISSED** [review]

**Review by colinbird — APPROVED** [review]

**Review by ashkarpetin — APPROVED** [review]

File: openapi/users.yaml [hunk 1]
```diff
@@ -118,9 +118,9 @@ tags:
       Payment cards are a type of payment instrument used for credit and debit card sales.
   - name: Organizations
     description: |
-      Organizations include the name and address of the entities related to your
-      account. An account may be multi-national, and support multiple
-      organizations. Note: Organizations are share between "Live" and "Sandbox".
+      Organization is an entity that represents you, or your company, as a merchant. You can have multiple organizations.
+
+      Note: Organizations are shared between "Live" and "Sandbox" environments.
```
Comment (adamaltman) [inline openapi/users.yaml:123]: What exactly is shared though besides the ID?

Comment (justnero) [reply openapi/users.yaml:123]: Whole Organization entity is shared. It includes organization address, phone numbers, emails, tax descriptor, time zone and a questionnaire.

Comment (adamaltman) [reply openapi/users.yaml:123]: That's upon organization creation. What about after that? Can they not diverge ever?

Comment (ashkarpetin) [reply openapi/users.yaml:123]: They can't at the moment, entity is synced between live and sandbox

**Pull request merged**